schema.json

```diff
@@ -62,13 +62,16 @@
 
 		"configManagementConfig": {
 			"type": "object",
+			"required": ["path"],
+			"properties": {
+				"path": {
+					"$ref": "#/definitions/relativePath"
+				}
+			},
+			"unevaluatedProperties": false,
```

Comment:
I wasn't familiar with this and had to look it up. Still having trouble wrapping my head around it, but I see you added test cases for this, so I am confident this is right.